tests/filesTest.js

```diff
@@ -523,4 +525,85 @@ describe('Main: Currently testing files management,', function () {
       done()
     })
   })
+
+  describe('TUS detection', function () {
+    var parseBodyStub
+    var xhr
+    var requests
+
+    beforeEach(function () {
+      xhr = sinon.useFakeXMLHttpRequest()
+      requests = []
+      xhr.onCreate = function (xhr) {
+        requests.push(xhr)
+      }
+      const dummyFileInfo1 = new FileInfo('dummy', 'dir', {})
+      const dummyFileInfo2 = new FileInfo('dummy2', 'dir', {})
+      parseBodyStub = sinon.stub(oc.helpers, '_parseBody').returns([dummyFileInfo1, dummyFileInfo2])
+    })
+
+    afterEach(function () {
+      parseBodyStub.restore()
+      xhr.restore()
+    })
+
+    it('returns TUS support information when TUS headers are set for a list call', function (done) {
+      const promise = oc.files.list('')
+      promise.then(entries => {
+        const tusSupport = entries[0].getTusSupport()
+        expect(tusSupport.resumable).toEqual('1.0.0')
```

Comment:
> I've re-read the spec and apparently only "Tus-Version" MUST be set and not the others.

Should this line still be here then or removed?

Reply:
I think we can keep it as I want to check whether the header is parsed when it's present (the test is only using a mock anyway)